test/e2e/upgrades/apps/deployments.go

```diff
@@ -21,15 +21,21 @@ import (
 
 	extensions "k8s.io/api/extensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
-	"k8s.io/kubernetes/pkg/util/version"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	//deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/upgrades"
 
 	. "github.com/onsi/ginkgo"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+const (
+	deploymentName     = "dp"
+	deploymentScaleNum = 2
```

Comment:
I don't think this should be `const`

Reply:
This should be `const` as we only intend to scale the deployment from 1 to a value _x_. The value _x_ will be immutable throughout the test. Refer replicaset upgrade test.

Reply:
See https://github.com/kubernetes/kubernetes/pull/53841#discussion_r145768471